clients/python/.gitignore

```diff
@@ -0,0 +1,66 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Do we really need everything in this file?

Reply:
it is auto-generated by the openapi-generator